clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,6 +30,7 @@ class GoogleConfigurationProperties {
     List<String> imageProjects
     ConsulConfig consul
     String userDataFile
+    List<String> regionsToIndex
```

Comment:
Two things here:

1). I think the name of the property here should match what's used for AWS (just `region`). I think it's OK to reference it as `regionsToIndex` on the `credentials` object. 

2). Like AWS, there should be a way to have a set of default regions for all GCE accounts, and then be able to override that set of defaults per account. You can do this piece in a later PR.